reference/forms/types/options/label_format.rst.inc

```diff
@@ -0,0 +1,46 @@
+label_format
+~~~~~~~~~~~~
+
+.. versionadded:: 2.6
+    The ``label_format`` option was introduced in Symfony 2.6.
+
+**type**: ``string`` **default**: ``null``
+
+Configures the string used as the label of the field, in case the ``label``
+option was not set. This is useful when using
+:ref:`keyword translation messages <translation-real-vs-keyword-messages>`.
+
+When using keyword translation messages as labels, you often end up having
+several keyword messages translating to the same label (e.g.
+``profile_address_street`` and ``invoice_address_street``). To avoid this,
```

Comment:
Got it now. Maybe we should reword the paragraph a bit then. At least to me both `profile_address_street` and `invoice_address_street` referred to a translation key and I did not understand them as strings derived from the form type.